### ISSUE TITLE
Use aria-hidden on banner designs

### ISSUE THREAD

**Issue by Elisa0122** [GCTC-NTGC/gc-digital-talent#6234]
## ❌ Current behaviour (issue)

VoiceOver identifies images without alt attribute in the following sample of pages:

1. [Job poster](https://uat-talentcloud.tbs-sct.gc.ca/en/browse/pools/136a3d08-de72-4b5c-b81f-a0dc2e91ecd5), 
2. [Job Application](https://uat-talentcloud.tbs-sct.gc.ca/en/browse/applications/4ccc0db3-e023-4a91-85b8-90c54364a9c9/apply), 
3. [GCKey explanation](https://uat-talentcloud.tbs-sct.gc.ca/en/login-info), 
4. [About me](https://uat-talentcloud.tbs-sct.gc.ca/en/users/528357e6-0ff7-4809-8f90-0bca8198aaf7/profile#about-section), 
5. [Diversity, equity, and inclusion](https://uat-talentcloud.tbs-sct.gc.ca/en/users/528357e6-0ff7-4809-8f90-0bca8198aaf7/profile/employment-equity/edit/?applicationId=ca3bce24-a7db-4741-8674-08d121f66702), 
6. [Edit skills and experience](https://uat-talentcloud.tbs-sct.gc.ca/en/users/528357e6-0ff7-4809-8f90-0bca8198aaf7/profile/experiences), 
7. [Work Experience](https://uat-talentcloud.tbs-sct.gc.ca/en/users/528357e6-0ff7-4809-8f90-0bca8198aaf7/profile/experiences/work/create), 
8. [Application submission page](https://uat-talentcloud.tbs-sct.gc.ca/en/browse/applications/4ccc0db3-e023-4a91-85b8-90c54364a9c9/submit)

**WCAG 2.1 Criteria:** 1.3.2: A correct reading sequence can be programmatically determined when the sequence in which content is presented affects its meaning.  

### How to reproduce

1. Go to one of the pages above
2. use apple Voice Over in chrome
3. Start reading by using VO + A
4. Confirm if the image conveys any essential information.

## ✔ Expected behaviour (fix)
The image is hidden in the banner on the pages above because it is just a decorative element. In turn allowing VoiceOver to programmatically determine and present a correct reading sequence to the user.  

### Acceptance Criteria
A set of assumptions which, when tested, verify that the accessibility requirement was met.

- [ ] Apple VoiceOver skips the image if the image contributes no meaning or context 
- [ ] Screen reader UI experience is not interrupted by an unidentified image tag
- [ ] use aria-hidden on banner designs

## Additional information (optional)

1. How does it impact users?
Having an image without a description or alt text can bother users who rely on assistive technologies, such as screen readers, to access the content on the web. Without a description, assistive technology cannot convey the meaning or context of the image to the user.

2. Related issues, suggestions how to fix
The image needs to be hidden in the banner on the pages above because it is just a decorative element.

**Comment by Elisa0122** [comment]
Thank you @petertgiles to take the bug over. Yes please use aria-hidden on the decorative banner image. Thanks
